config/7.12.0/pim/process-migration.yaml

```diff
@@ -88,20 +95,104 @@ processMigration:
+                  public:
+                    paths: /q/health/*
+                    policy: permit
+                    methods: GET
+            security:
+              users:
+                file:
+                  realm-name: pim-file
+                  enabled: true
+                  plain-text: false
+                  users: /opt/rhpam-process-migration/quarkus-app/config/application-users.properties
+                  roles: /opt/rhpam-process-migration/quarkus-app/config/application-roles.properties
+              jdbc:
+                realm-name: pim-jdbc
+                enabled: true
+                principal-query:
+                  sql: SELECT u.password, u.role FROM users u WHERE u.username=?
+              ldap:
+                realm-name: pim-ldap
+                enabled: true
+                dir-context:
+                  url: ldap://override-when-needed
+                identity-mapping:
+                  search-base-dn: ou=users,o=YourCompany,c=ES
+            # Flyway to create PIM schema
+            flyway:
+              connect-retries: 10
+              table: flyway_pim_history
+              migrate-at-start: true
+              baseline-on-migrate: true
+              baseline-version: 1.0
+              baseline-description: PimDB
+              sql-migration-prefix: h2
+            # Quartz configuration
+            quartz:
+              store-type: jdbc-cmt
+              start-mode: forced
+            resteasy:
+              path: /rest
+            datasource:
+              db-kind: h2
+              jdbc:
+                url: jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+              username: sa
+              password: sa
+            hibernate-orm:
+              database:
+                generation: validate
+          pim:
+            auth-method: file
           kieservers:
             #[[range $index, $Map := .ProcessMigration.KieServerClients]]
             - host: [[.Host]]
               username: [[.Username]]
               password: [[.Password]]
             #[[end]]
+        application-users.properties: |-
+          # set the following spec to autogenerated this file content, if you want to provide your own properties file
+          # override this file's content.
+          # spec:
+          #  objects:
+          #    processMigration:
+          #      username: pimAdmin
+          #      password: somePasswordThatWillBeConvertedToMD5
+          # to generate password use:  echo -n "username:pim-file:password" | openssl md5
+          # pim-file is the realm defined on the application.yaml config-map;
+          [[.ProcessMigration.Username]]=[[.ProcessMigration.Password]]
+        application-roles.properties: |-
+          [[.ProcessMigration.Username]]=admin
   services:
     - spec:
         ports:
           - name: http
             port: 8080
             protocol: TCP
             targetPort: 8080
+          - name: https
```

Comment:
Why adding the https port? I don't think it's enabled

Reply:
I've removed already locally.